docs/contributing/api-reference.md

```
@@ -1,14 +1,254 @@
+  "provider": "openai_realtime",
+  "transport": "external_media"
+}
+```
+
+### Example: Prometheus Metrics
+
+```bash
+curl http://localhost:15000/metrics
+```
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)